src/main/java/com/mongodb/spark/sql/connector/config/WriteConfig.java

```diff
@@ -179,6 +179,21 @@ public String toString() {
 
   private static final boolean CONVERT_JSON_DEFAULT = false;
 
+  /**
+   * Keep null values in the BSON document.
+   *
+   * <p>Configuration: {@value}
+   *
+   * <p>Default: {@value IGNORE_NONE_DEFAULT}
+   *
+   * <p>If true, will store a field in the database only if it is present.
+   *
+   * @since 10.1
+   */
+  public static final String IGNORE_NONE_CONFIG = "ignoreNone";
```

Comment:
nit: As this is a java code base and `None` is a scala type, I'll update to use `null` instead.